### PR TITLE
github action: add presubmit and postsubmit

### PR DESCRIPTION
**Pull request by ycyaoxdu**
github action: add presubmit and postsubmit
makefile: add copyright header check and lint

Signed-off-by: ycyaoxdu <yaoyuchen0626@163.com>

### CODE REVIEW

File: .copyrightignore [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+pkg/apiserver/options/authorization.go
+pkg/apiserver/options/embeddedetcd.go
+pkg/apiserver/options/options.go
+pkg/apiserver/options/serving.go
+pkg/controllers/kubecontroller/bootstrap.go
+pkg/controllers/kubecontroller/import_known_versions.go
+pkg/controllers/kubecontroller/options/csrsigningcontroller.go
+pkg/controllers/kubecontroller/options/garbagecollectorcontroller.go
+pkg/controllers/kubecontroller/options/namespacecontroller.go
+pkg/etcd/etcd.go
```
Comment (clyang82) [inline .copyrightignore:20]: Can we ignore `.yaml` file for copyright check? Is it necessary to add check for yaml file?

Comment (ycyaoxdu) [reply .copyrightignore:21]: done

**Comment by clyang82** [comment]
💯 
/LGTM

**Pull request merged**